### PR TITLE
SISRP-40864 - Terms in Attendance display missing from profile in CalCentral

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-40864

### CODE REVIEW

File: src/assets/templates/widgets/academics/college_and_level.html [hunk 1]
```diff
@@ -26,5 +26,6 @@
     </tbody>
   </table>
 </div>
+<div>Terms In Attendance: <span data-ng-bind="collegeAndLevel.termsInAttendance"></span></div>
```
Comment (davidlimcheng) [inline src/assets/templates/widgets/academics/college_and_level.html]: @redconfetti I believe this needs to be removed

**Comment by redconfetti** [comment]
@davidlimcheng - Amended

**Comment by davidlimcheng** [comment]
@redconfetti Looks like Travis got stuck, i'll go ahead and restart it.

**Comment by lyttam** [comment]
@redconfetti Thanks for the fix!

**Pull request merged**